README.md

```diff
@@ -0,0 +1,108 @@
+# Perform
+
+Easy dependency injection for storyboard segues.
+
+```swift
+import Perform
+
+// ...
+
+func tableView(_ tableView: UITableView, didSelectRowAt indexPath: NSIndexPath) {
+  let task = taskList[indexPath.row]
+  perform(.showTaskDetails) {
+    $0.task = task
```

Comment:
It's not clear to me what type this `$0` is, might be nice to add an annotation or something describing this